lib/src/test/java/com/knuddels/jtokkit/reference/Cl100kBaseTest.java

```diff
@@ -19,16 +21,23 @@
 
 public class Cl100kBaseTest {
     public static final String PUNCTUATION = "'\".,?!:()";
-    private static final String LETTERS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZő你好ſ ½";
-    private static final String NUMBERS = "0123456789½";
+    private static final String LETTERS = generateUnicodeCategoryString(Cl100kParser::isLetter);
```

Comment:
we don't have to list them out anymore, just make sure every special category is generated in the automated test